pkgs/development/tools/jnv/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, autoconf
+, automake
+, clang
+, libclang
+, libtool
+}:
+rustPlatform.buildRustPackage rec
+{
+  pname = "jnv";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "ynqa";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-LN42UyXOStwgj/bbHl2MK1jTsDACTjDo+QaoZ9FzurA=";
+  };
+
+  cargoHash = "sha256-ZCYDlvKSuEONPSz/TS00mYolPnMYxsOwE9a0vK36UQQ=";
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    clang
+    libclang
+    libtool
+  ];
+
+  cargoPatches = [
```

Comment:
And don't forget about the cargoLock.lockFile attribute!

https://github.com/NixOS/nixpkgs/blob/e671d3bbbda9b2bec8fa119aa1ff46499118ef17/doc/languages-frameworks/rust.section.md#importing-a-cargolock-file-importing-a-cargolock-file